modin/data_management/functions/default_methods/groupby_default.py

```diff
@@ -195,16 +211,28 @@ def fn(
             map_args,
             numeric_only=True,
             drop=False,
+            selection=None,
             **kwargs
         ):
             """Group DataFrame and apply aggregation function to each group."""
+            if selection is not None:
+                selection = selection if is_list_like(selection) else (selection,)
             if not isinstance(by, (pandas.Series, pandas.DataFrame)):
                 by = cls.validate_by(by)
-                return agg_func(
-                    df.groupby(by=by, axis=axis, **groupby_args), **map_args
-                )
-
-            if numeric_only:
+                grp = df.groupby(by=by, axis=axis, **groupby_args)
+                if selection is not None:
+                    grp = grp[selection]
+                result = agg_func(grp, **map_args)
+                if (
```

Comment:
same as [here](https://github.com/modin-project/modin/pull/3298/files#r687873043)

Reply:
moved this logic to the [`_veryfy_aggregation_result`](https://github.com/YarShev/modin/blob/9be84a9f8deaf70819db5b437776441fbb971d5c/modin/pandas/groupby.py#L1154) method, located at the front-end